projects/adrv9026/zcu102/system_project.tcl

```diff
@@ -0,0 +1,28 @@
+###############################################################################
+## Copyright (C) 2023 Analog Devices, Inc. All rights reserved.
+### SPDX short identifier: ADIBSD
+###############################################################################
+
+source ../../../scripts/adi_env.tcl
+source $ad_hdl_dir/projects/scripts/adi_project_xilinx.tcl
+source $ad_hdl_dir/projects/scripts/adi_board.tcl
+
+adi_project adrv9026_zcu102 0 [list \
+  TX_JESD_M       [get_env_param TX_JESD_M       8 ] \
+  TX_JESD_L       [get_env_param TX_JESD_L       4 ] \
+  TX_JESD_S       [get_env_param TX_JESD_S       1 ] \
+  RX_JESD_M       [get_env_param RX_JESD_M       8 ] \
+  RX_JESD_L       [get_env_param RX_JESD_L       4 ] \
+  RX_JESD_S       [get_env_param RX_JESD_S       1 ] \
+  RX_OS_JESD_M    [get_env_param RX_OS_JESD_M    2 ] \
+  RX_OS_JESD_L    [get_env_param RX_OS_JESD_L    2 ] \
+  RX_OS_JESD_S    [get_env_param RX_OS_JESD_S    1 ] \
```

Comment:
These don't seem to be used, so they should be removed

Reply:
Removed